### DISCUSSION TITLE
provision QA env failure

### DISCUSSION THREAD

**Discussion by mdminhaj88** [opencrvs/opencrvs-core#7846] [Technical Q&A] [answered]
### OpenCRVS Core version

1.5.0

### OpenCRVS Countryconfig version

1.5.0

### Your operating system and version

Ubuntu 22.04

### Your node version

v16.20.0

### Which of these describes your project best?

- [X] Testing OpenCRVS for business development, educational purposes or for personal interest.
- [ ] A research project to evaluate, configure and test OpenCRVS as a possible solution for a government, an active brief or proposal.
- [ ] Actively installing and configuring OpenCRVS Core for production use in a country.
- [ ] Customising OpenCRVS Core code in production in a country

**Comment by mdminhaj88** [comment]
provision QA env is failing at this point 
![image](https://github.com/user-attachments/assets/923dd98e-ba98-496b-bcbd-2b86036498bd)

**Comment by Zangetsu101** [comment, marked as answer]
Hi @mdminhaj88. I noticed that you are using Ubuntu 22.04, could you try again after upgrading to 24.04 please as that's the version we've tested with. I'm guessing it's something to do with the python version in your current system as mentioned here: https://forum.ansible.com/t/error-saying-customhttpsconnection-object-has-no-attribute-cert-file/7390
